extensions/libarchive/MergeContent.cpp

```diff
@@ -342,6 +362,56 @@ std::shared_ptr<core::FlowFile> ZipMerge::merge(core::ProcessContext *context, c
   return flowFile;
 }
 
+void AttributeMerger::mergeAttributes() {
+  std::map<std::string, std::string> commonAttributes = getCommonAttributes();
```

Comment:
we could do without the local variable, also I would rename `getCommonAttributes` to `getMergedAttributes` (or something else) as "common" has a specific meaning in this context, i.e. attributes that have the same value in all flowFiles, (similarly for the parameter `commonAttributes` in the `processFlowFile` methods)